doc/docs.md

```diff
@@ -348,6 +348,8 @@ Global (module level) variables are not allowed. There's no global state in V.
 For consistency across different code bases, all variable and function names
 must use the `snake_case` style, as opposed to type names, which must use `PascalCase`.
 
+By default V does not allow **global variables**. See more [details](#global-variables).
+
```

Comment:
Line 346 is:
`Global (module level) variables are not allowed. There's no global state in V.`
I think it should be deleted, replaced by your new line.